apps/docs/src/app/core/component-docs/calendar/calendar-docs.component.ts

```diff
@@ -63,6 +69,42 @@ myDisableFunction = function(d: FdDate): boolean {
     return d.getTimeStamp() > firstDay.getTimeStamp() && d.getTimeStamp() < lastDay.getTimeStamp()
 }`;
 
+
+    exampleSpecialDays = `Example Special Days Functions:
+
+// Mark weekends with special day number 10
```

Comment:
This section of the docs is a little confusing, I thought that the example would use this rule.  I think the better option would be to use the more complex rule at the bottom in the example

Reply:
Done, take a look at new section